### PR TITLE
Farsi Translation

### PR DESCRIPTION
**Pull request by esfand-r**
@mvalipour @amirkarimi please kindly review and invite other folks you know who might be interested in a review. I'm not sure I have managed to get the formatting completely right with ```<p dir="rtl">``` ! Unfortunately those html tags don't work properly in "Files Changed" tab but otherwise I think they should be ok. There was quite a bit of annoyance especially when mixing RTL with LTR together.

### CODE REVIEW

**Comment by amirkarimi** [comment]
It seems that when you use HTML tags, markdown engine switches to HTML mode and ignores markdown syntax.

File: README.fa.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+
+<h2 dir="rtl">
+مانیفست واكنشی
```
Comment (mvalipour) [inline README.fa.md]: I think `بیانیه` is a better translation for "Manifesto"

Comment (mvalipour) [inline README.fa.md]: Also `واکنشگر` should be used consistently in the document?

**Comment by esfand-r** [comment]
@amirkarimi that seems to be the case, I updated to use only html tags which is rather unfortunate. The end result looks like this https://github.com/esfand-r/testing-rtl/blob/master/README.md

File: README.fa.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+<p dir="rtl">
+
+شرکت های فعال درحوزه های مختلف به طور مستقل الگوهایی برای ساختن نرم افزار کشف می کنند که به هم شبیه هستند. این نرم افزارها دارای بنیادی قویتر، ترمیم پذیرتر و با قابلیت انعطاف بیشتر هستند و درموقعیت بهتری برای پاسخگویی به نیازهای امروزی قرار دارند.
+
+</p>
+
+<p dir="rtl">
+
+نیازهای برنامه‌های كاربردی در سال‌های اخیر به طور چشمگیری تغییر كرده‌اند. تنها چند سال قبل یك برنامه كاربردی بزرگ دارای ده‌ها سرور، زمان پاسخگویی در حد چند ثانیه، و زمان نگهداری آفلاین در حد ساعت، و داده‌هایی در حد چند گیگابایت بود. امروزه برنامه‌های كاربردی بر روی همه چیز از وسیله‌های همراه گرفته تا خوشه های مبتنی بر كلاود مستقر هستند كه هزاران پردازنده چند هسته‌ای را به كار می‌گیرند. كاربران خواستار زمان پاسخگویی در حد هزارم ثانیه و زمان دسترسی ۱۰۰ درصد هستند. داده‌ها در مقیاس پتابایت اندازه گیری می شوند. معماری نرم افزار های دیروز، دیگر پاسخگوی نیازهای امروز نیستند.
```
Comment (ehsanrohani) [inline README.fa.md]: Suggestion: memari e narmafzar haye dirooz digarpasokhgooye niazhaye emooze nistand.Manabe morede niaz barname haye karbordi dar sal haye akhir be tore cheshmgiri taghir karde and. Be onvane mesal dar salhaye akhir yek barnameye karbordi niazmand chandin server, zamane pasokhgooyie kootah (dar had chand sanie) va negahdarie toolani modat bood. Dar halike ke emrooze barnamehaye karbordi az telephone haye hamrah gerefte ta khooshe haye mobtani bar cluster ba hezaran pardazandeye hasteyi mostaghar boode, zamane pasokhgooyi be hezarom sanie kahesh yafte va hata dade ha dar had e peta byte andaze giri mishabvand.

Comment (mvalipour) [inline README.fa.md]: Please propose your suggestion in Farsi script.

File: README.fa.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+</p>
+
+<p dir="rtl">
+
+ما معتقدیم که در معماری سیستم به یک رویکرد منسجم احتیاج است، و معتقدیم که تمام جنبه های لازم در حال حاضر به صورت جداگانه به رسمیت شناخته شده اند: ما سیستم هایی می خواهیم که پاسخگو ، ترمیم‌ پذیر، منعطف و پیام-گرا باشند. ما این سیستم ها را واکنشگر می نامیم.
```
Comment (ehsanrohani) [inline README.fa.md]: ا بر این باوریم که طراحی سیستم نیازمند یک رویکرد منسجم بوده و تمامی اجزای تشکیل دهنده‌ی این رویکرد به طور مجزا و جداگانه شناخته شده اند: هدف طراحی سیستمی پاسخگو , ترمیم پذیر, منعطف و پیام گراست. ما چنین سیستمی را واکنشگر مینامیم.

File: README.fa.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+</p>
+
+<p dir="rtl">
+
+سیستم هایی که به عنوان سیستم های واکنشگرساخته شده اند قابل انعطاف تر، با پیوند سست و   <a href="/glossary#Scalability)">میزان پذیر</a> هستند. این امر ساختن آنها را آسان تر و آنها را مستعد تغییر می کند. آنها به طور قابل توجهی بیشتر تحمل <a href="/glossary#Failure">خرابی</a> را دارند و زمانی که خرابی رخ می دهد، با آنها با ظرافت و نه مانند فاجعه برخورد میکنند. سیستم های واکنشگر  بسیار پاسخگو هستند و برای <a href="/glossary#User">کاربران</a> بازخورد تعاملی موثری فراهم می کنند.
```
Comment (ehsanrohani) [inline README.fa.md]: سیستم هایی که به عنوان سیستم های واکنشگر شناخته شده اند در مقایسه با دیگر سیستم ها در خصوصیات زیر برتری دارند;

- همراهی آزاد‌انه 

- مقیاس پذیری بالا

- انطاف پذیری بیشتر 

- طراحی آسانتر

- قابلیت تغییر پذیری

- تحمل خطای بالاتر ( خطا در سیستم منتهی به فاجهه نخواهد بود و با درایت و ظرافت خاصی مورد بررسی قرار میگیرند)

- واکنش پذیری بیشتر

File: README.fa.md (hunk 2 above)
Comment (esfand-r) [inline README.fa.md]: Thanks. There are some changes in the structure of the paragraph that I don't personally think is right. But if another reviewer agrees, ill make some changes. for example the suggestion says 
به عنوان مثال در سال‌های اخیر

but the English version does not have it. The meaning has also slightly changed. In general too much changes to structure and wording that does not necessarily add to the clarity of the technical text. Cheers

File: README.fa.md (hunk 4 above)
Comment (esfand-r) [inline README.fa.md]: The suggestion is to make a bullet list in this paragraph? That I'm afraid won't make sense to do. In that case you can make a suggestion on the English version that can trigger such change in the translation too.

File: README.fa.md (hunk 3 above)
Comment (esfand-r) [inline README.fa.md]: done thanks

File: README.fa.md (hunk 1 above)
Comment (esfand-r) [inline README.fa.md]: done thanks

File: README.fa.md (hunk 2 above)
Comment (mvalipour) [inline README.fa.md]: Agreed. Although I'm up for deep translation (as opposed to word-by-word), it is also important that we don't drop/add extra semantic from the original copy. -- after all this is a 'manifesto'! :)

**Comment by ehsanrohani** [comment]
LGTM

**Comment by esfand-r** [comment]
@jboner please kindly consider this one for merge when you have some time. Thanks

**Comment by jboner** [comment]
Thanks a lot for the hard work.

**Comment by esfand-r** [comment]
Cheers

**Comment by rasummer** [comment]
@esfand-r could you be kind enough to translate the messages file in the following PR https://github.com/reactivemanifesto/website-manifesto/pull/54/files#diff-0d6c1bbbe074a265266a8bff4f41baf0

Once done, we'll then be able to publish the translation to the public website.

Many thanks, Richard

**Comment by esfand-r** [comment]
@rasummer absolutely. I will make a PR very soon. Cheers.

**Comment by jboner** [comment]
@esfand-r What's the status of the PR?

**Comment by rasummer** [comment]
@jboner the Messages file over at the website-manifesto was merged and deployed a while back, we can consider this complete. Thanks @esfand-r

**Comment by jboner** [comment]
Great. Thanks.

**Pull request merged**